app/services/salary_reporting_service.rb

```diff
@@ -0,0 +1,46 @@
+# Responsible for database reporting
+class SalaryReportingService
+  def current_salary_by_graduation_year
+    sql = <<-SQL
+      SELECT
+        profiles.graduation_year AS graduation_year,
+        COUNT(salaries.id) AS submission_count,
+        CAST(AVG(salaries.amount) AS INTEGER) AS average_salary,
+        MIN(salaries.amount) AS minimum_salary,
+        MAX(salaries.amount) AS maximum_salary
+      FROM
+        salaries
+        JOIN users ON salaries.user_id = users.id
+        JOIN profiles ON users.id = profiles.user_id
+      WHERE
+        salaries.current_salary = true
+      GROUP BY
+        1
+      ORDER BY
+        1
```

Comment:
Could we update these to be explicit? I see that it's just grouping / ordering by the first column of the select clause, but it'd be nice not to rely on this. That way if things change in future, we know it'll still be grouping / ordering by the same column we originally set it to. It's more readable, too.

Reply:
Good call.

Reply:
🎉 Thanks!